dotnet-desktop-guide/net/winforms/printing/overview.md

```diff
@@ -0,0 +1,191 @@
+---
+title: "PrintDialog Component Overview"
+ms.description: "Learn about print related settings in Windows-based applications."
+ms.date: "04/13/2022"
```

Comment:
This needs to have the code language metadata added to it so that the code switcher appears in the article
```suggestion
ms.date: "04/13/2022"
dev_langs:
  - "csharp"
  - "vb"
```

Reply:
done